renormalizer/mps/gs.py

```diff
@@ -363,6 +363,15 @@ def get_ham_direct(
     return ham
 
 
+def sign_fix(c, nroots):
+    if nroots > 1:
+        if isinstance(c, list):
+            return [ci / np.sign(np.max(np.abs(ci))) for ci in c]
```

Comment:
1. Taking abs loses the sign information
2. Add abs for the following branches as well?

Reply:
1. yea you are right. my mistake
2. what do you mean the following branches?

Reply:
when c is not a list and when nroots == 1

Reply:
got it!